staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_status.go

```diff
@@ -228,4 +230,10 @@ func (o *RolloutStatusOptions) Run() error {
 			return err
 		})
 	})
+
+	if !resourceFound {
+		fmt.Fprintf(o.Out, "No resources found in %s namespace.\n", o.Namespace)
```

Comment:
I think, this message should print to ErrOut instead Out as in https://github.com/kubernetes/kubernetes/blob/00c30941260a27e6929aef84c7fdbc8f1508518c/staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go#L286